R/format_KIL.R

```diff
@@ -0,0 +1,657 @@
+                  .data$AvgTarsus, .data$NumberChicksTarsus,
+                  .data$OriginalTarsusMethod, .data$ExperimentID)
+
+
+  Brood_data <-
+    Brood_data_til92 %>%
+    dplyr::bind_rows(Brood_data_95)
+
+  return(Brood_data)
+
+}
+
+
+
+#### CAPTURE DATA
+
+#### SOLVE: There is no date available, only the year.
+#### Create fake date?
+# CaptureDate = as.Date(paste0(BreedingSeason, "-05-01"))
+
+
+create_capture_KIL <- function(brood_data_til92, adults_data_til92,
+                               chicks_data_til92, adults_data_95) {
+
+  ### Subset brood data to get location (nestbox) ID
+  brood_data_til92_temp <-
+    brood_data_til92 %>%
+    dplyr::select(BroodId, Species, PopID, BreedingSeason, LocationID)
+
+
+  #### Adults
+  #### Data from 1971 to 1992
+  capture_adults_til92 <-
+    adults_data_til92 %>%
+    dplyr::mutate(RingAge = "adult",
+                  Sex_observed = .data$Sex,
+                  BreedingSeason = as.integer(.data$RingDate),
+                  #### Create the date, as there is no information
+                  CaptureDate = as.Date(paste0(BreedingSeason, "-05-01")),
+                  CaptureTime = NA_character_,
+                  ObserverID  = NA_character_,
+                  CaptureAlive = TRUE,
+                  ReleaseAlive = .data$CaptureAlive,
+                  CapturePopID = .data$PopID,
+                  CapturePlot  = NA_character_,
+                  ReleasePopID = ifelse(ReleaseAlive == TRUE, CapturePopID, NA_character_),
+                  ReleasePlot  = ifelse(ReleaseAlive == TRUE, CapturePlot, NA_character_),
+                  Mass = NA_real_,
+                  Tarsus = NA_real_,
+                  #### ASK DATA OWNER
+                  OriginalTarsusMethod = NA_character_,
+                  #### ASK DATA OWNER what do the codes mean? Age or Euring codes?
+                  # Age_observed = case_when()
+                  ChickAge = NA_integer_,
+                  ExperimentID = NA_character_)
+
+
+
+  #### Chicks
+  #### Data from 1971 to 1992
+  capture_chicks_til92 <-
+    chicks_data_til92 %>%
+    dplyr::mutate(RingAge = "chick",
+                  Sex_observed = .data$Sex,
+                  BreedingSeason = as.integer(.data$RingDate),
+                  #### Create the date, as there is no information
+                  CaptureDate = as.Date(paste0(BreedingSeason, "-05-01")),
+                  CaptureTime = NA_character_,
+                  ObserverID  = NA_character_,
+                  CaptureAlive = TRUE,
+                  ReleaseAlive = .data$CaptureAlive,
+                  CapturePopID = .data$PopID,
+                  CapturePlot  = NA_character_,
+                  ReleasePopID = ifelse(ReleaseAlive == TRUE, CapturePopID, NA_character_),
+                  ReleasePlot  = ifelse(ReleaseAlive == TRUE, CapturePlot, NA_character_),
+                  Mass = .data$Mass,
+                  Tarsus = .data$Tarsus*0.777 + 6.158,
```

Comment:
We have an explicit `convert_tarsus()` function that we can use here 😁 

```suggestion
                  Tarsus = convert_tarsus(.data$Tarsus, method = "Standard"),
```

Reply:
Great, I was not aware of that :)